domains/certificates/Query.API/API.AppTests/Infrastructure/RabbitMqContainer.cs

```diff
@@ -0,0 +1,32 @@
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Containers;
+using Xunit;
+
+namespace API.IntegrationTest.Infrastructure;
+
+public class RabbitMqContainer : IAsyncLifetime
+{
+    private readonly RabbitMqTestcontainer testContainer;
+
+    public RabbitMqContainer() =>
+        testContainer = new TestcontainersBuilder<RabbitMqTestcontainer>()
+            .WithMessageBroker(new RabbitMqTestcontainerConfiguration
+            {
+                Username = "guest",
+                Password = "guest"
```

Comment:
Since username and password is exposed as properties, is it necessary to set them here? Will they be null otherwise?

Reply:
Yes it is necessary, else testcontainer will look for some env vars that doesnt exists